sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1182,6 +1182,44 @@ Exceptions:
 
 |===
 
+==== New Event Member Functions
+
+Table {counter: tableNumber}. Additional member functions of the `sycl::event` class.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+template <typename Param>
+typename Param::return_type 
+event::ext_oneapi_get_profiling_info(node Node) const;
+----
+
+| Queries the profiling information of a SYCL Graph node for the graph
+  execution associated with this SYCL event. If the requested info is
+  not available when this member function is called due to incompletion of
+  graph execution associated with the event, then the call to this member
+  function will block until the requested info is available.
```

Comment:
@EwanC asked in #12838 how `ext_oneapi_get_profiling_info` should relate to the new `submit_profiling_tag` API.

I think the graph extension could add a new member function to `command_graph`:

```
node add_profiling_tag();
```

This function would fail unless the graph's device had the aspect `ext_oneapi_queue_profiling_tag`.  Once the application uses this API to add a profiling tag to the graph, it can get the profiling information later via `ext_oneapi_get_profiling_info`.